Application/EdFi.Ods.SandboxAdmin/Views/Vendor/Index.cshtml

```diff
@@ -0,0 +1,193 @@
+﻿<!--
+  SPDX-License-Identifier: Apache-2.0
+  Licensed to the Ed-Fi Alliance under one or more agreements.
+  The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+  See the LICENSE and NOTICES files in the project root for more information.
+-->
+
+@{
+    ViewBag.Title = "Sandbox Administration Portal";
+    Layout = "~/Views/Shared/Layout.cshtml";
+}
+
+<script src="~/js/vendor.js"></script>
+
+<div class="row">
+    <div class="jumbotron">
+        <h3>Sandbox Administration Portal for the Ed-Fi ODS API</h3>
+        <p class="lead">
+            This portal enables you to manage sandboxes for the Ed-Fi ODS API, including the keys and secrets for accessing these sandboxes.
+        </p>
+    </div>
+</div>
+
+<div class="row">
+    <div class="alert alert-danger" data-bind="visible: error">
+        <strong>Error:</strong> <span data-bind="text: error"></span>
+    </div>
+</div>
+
+<div class="container-fluid">
+    <div class="row">
+        <div class="col-md-12">
+            <div class="float-md-right">
+                <button id="add-application" class="btn btn-primary" data-bind="enable: canAddMoreVendors, click: addVendorClicked">Add Vendor</button>
+            </div>
+    <fieldset>
+        <legend>Existing Vendors</legend>
+        <table class="table table-hover" id="clients-table">
+            <thead>
+                <tr>
+                    <th></th>
+                    <th>Vendor Name</th>
+                    <th>NamespacePrefix</th>
```

Comment:
Please add a space between the words so that it ends up being `Namespace Prefix`.

Reply:
Fixed it .Thanks